ush/generate_FV3LAM_wflow.py

```diff
@@ -494,12 +489,211 @@ def generate_FV3LAM_wflow(ushdir, logfile: str = "log.generate_FV3LAM_wflow", de
+    # of the surface climatology files depend on the CRES parameter (which is
+    # the C-resolution of the grid), and this parameter is in most workflow
+    # configurations is not known until the grid is created.
+    #
+    if not expt_config['rocoto']['tasks'].get('task_make_grid'):
+
+        set_FV3nml_sfc_climo_filenames()
+
+    #
+    # -----------------------------------------------------------------------
+    #
+    # Generate namelist for surface cycle
+    #  Deleted in newer RRFS_dev version so here for testing only
+    #
+    # -----------------------------------------------------------------------
+    #
+    if DO_SURFACE_CYCLE:
+        if SDF_USES_RUC_LSM:
+            lsoil=9
+        settings = {}
+        settings["gfs_physics_nml"] = {
+            "lsoil": lsoil or None
+            #"fh_dfi_radar": FH_DFI_RADAR # commented out untile develop gets radar tten code
+        }
+
+        settings_str = cfg_to_yaml_str(settings)
+        #
+        # populate the namelist file
+        #
+        try:
+            set_namelist(
+                [
+                    "-q",
+                    "-n",
+                    FV3_NML_FP,
+                    "-u",
+                    settings_str,
+                    "-o",
+                    FV3_NML_CYCSFC_FP,
+                ]
+            )
+        except:
+            logging.exception(
+                dedent(
+                    f"""
+                    Call to python script set_namelist.py to generate an FV3 namelist file
+                    failed.  Parameters passed to this script are:
+                      Full path to output namelist file:
+                        FV3_NML_FP = '{FV3_NML_FP}'
+                      Full path to output namelist file for DA:
+                        FV3_NML_RESTART_FP = '{FV3_NML_CYCSFC_FP}'
+                      Namelist settings specified on command line:\n
+                        settings =\n\n"""
+                )
+                + settings_str
+            )
+      
+    #
+    # -----------------------------------------------------------------------
+    #
+    # Generate namelist for DA cycle
+    #
+    # -----------------------------------------------------------------------
+    #
+    if DO_DACYCLE or DO_ENKFUPDATE:
+
+        if SDF_USES_RUC_LSM:
+            lsoil = 9
+
+        lupdatebc = False
+        if DO_UPDATE_BC:
+            lupdatebc = False   # not ready for setting this to true yet
+        
+        settings = {}
+        settings["fv_core_nml"] = {
+            "external_ic": False,
+            "make_nh": False,
+            "na_init": 0,
+            "nggps_ic": False,
+            "mountain": True,
+            "regional_bcs_from_gsi": lupdatebc,
+            "warm_start": True,
+        }
+        settings["gfs_physics_nml"] = {
+            "lsoil": lsoil or None
+            #"fh_dfi_radar": FH_DFI_RADAR # commented out untile develop gets radar tten code
```

Comment:
As per a conversation last week, I don't know that we will introduce radar tten to SRW.

Reply:
Removed